sotodlib/mapmaking/ml_mapmaker.py

```diff
@@ -87,7 +87,7 @@ def add_obs(self, id, obs, deslope=True, noise_model=None, signal_estimate=None)
         tod = nmat.apply(tod)
         # Add the observation to each of our signals
         for signal in self.signals:
-            signal.add_obs(id, obs, nmat, tod, glitch_flags=self.glitch_flags_path)
+            signal.add_obs(id, obs, nmat, tod)
```

Comment:
Can you include the `cut_fields=self.glitch_flags_path` in this call?
SO data have their `glitch_flags` under `flags`, while ACT has it under the observation.

All the `glitch_flags` inputs where for that reason.